api/http-request-practice.md

```diff
@@ -96,32 +92,9 @@ Why was it the same? In our case, Postman generated a set of sensible defaults.
 
 - If Postman can generate code for `curl` and other tools, what does that tell us about Postman?
 
-#### XMLHttpRequest - Time API
-
-Now let's go into the browser world, which is more useful for creating web apps.
-
-First, we will use the original interface, `XMLHttpRequest` which was added to JavaScript to allow the original background requests commonly called _AJAX_. It is a little awkward to use.
-
-```javascript
-function reqListener () {
-  console.log(this.responseText);
-}
-
-var oReq = new XMLHttpRequest();
-oReq.addEventListener("load", reqListener);
-oReq.open("GET", "https://worldtimeapi.org/api/ip");
-oReq.send();
-```
-
-If correct, you should see the result:
-
-```javascript
-Object { week_number: 47, utc_offset: "-08:00", utc_datetime: "2019-11-20T19:17:12.777523+00:00", unixtime: 1574277432, timezone: "America/Los_Angeles", raw_offset: -28800, dst_until: null, dst_offset: 0, dst_from: null, dst: false, … }
-```
-
 #### `window.fetch` - Time API
 
-`XMLHttpRequest`, which you just learned about, has a complicated interface which led most engineers to wrap it in a friendlier syntax. jQuery was the most common one because it also gave developers lots of other helpful utilities.
+`XMLHttpRequest` has a complicated interface which led most engineers to wrap it in a friendlier syntax. jQuery was the most common one because it also gave developers lots of other helpful utilities.
```

Comment:
There is no need to keep this sentence, as if we don't have that in the file then what's the need to mention it.

Reply:
@Kundan28 I deleted the sentence.